### PR TITLE
Additions to GDSC Map for DSC Lead- Vasundhara Shukla

### PR DESCRIPTION
**Pull request by Vasundharashukla**
Fixes #

## Proposed Changes

-
-
-

### CODE REVIEW

File: src/components/json/worldData.js [hunk 1]
```diff
@@ -559,6 +560,21 @@ const data = [
 		linkedin: "https://www.linkedin.com/in/venky-jayasurya/",
 		twitter: "https://twitter.com/venky_jayasurya/",
 	},
+	{
+		longitude: 80.886337,
+		latitude: 26.846300,
+		batch: 2020,
+		photo: Vasundharashukla,
+		region: "Asia",
+		skills: ["XR", "Web Dev", "Cloud", "ML"],
```
Comment (aravindvnair99) [inline src/components/json/worldData.js:569]:
```suggestion
		skills: ["XR", "Web Dev", "Cloud", "AI/ML"],
```

**Review by aravindvnair99 — CHANGES_REQUESTED** [review]
@Vasundharashukla Please only use skills mentioned in the [guidelines](https://github.com/abhinavsri360/GDSC-Lead-Map/blob/main/CONTRIBUTING.md).

**Review by abhinavsri360 — APPROVED** [review]
Looks good to me now.
Thanks for adding yourself!

**Comment by abhinavsri360** [comment]
😂 @aravindvnair99 @Vasundharashukla 
I made that change, I added the suggestion as a commit to the PR changes :))

**Comment by Vasundharashukla** [comment]
*Face palm* @abhinavsri360 @aravindvnair99

**Comment by abhinavsri360** [comment]
🤦🏻‍♂️

**Comment by aravindvnair99** [comment]
> 😂 @aravindvnair99 @Vasundharashukla
> I made that change, I added the suggestion as a commit to the PR changes :))

@abhinavsri360 Yeah I was just clarifying so that @Vasundharashukla understands how the suggestion feature works. 😅

**Pull request merged**